### ISSUE TITLE
Decrypting titlekey for DON'T CRASH GO fails

### ISSUE THREAD

**Issue by enderghast13** [d0k3/GodMode9#93]
I had legitimately purchased this title from the eshop a while ago and when when trying to read the decrypted titlekey by selecting "Build decTitleKeys.bin" on my ticket.db, its titlekey did not appear in the file. Manually selecting its ticket from the mounted ticket.db image and selecting "Build decTitleKeys.bin" also fails. It can, however build an encTitleKeys.bin from the ticket, but it cannot build the resulting file into a decTitleKeys.bin file. Every other ticket I've tried to build into decTitleKeys.bin has succeeded thus far except for this one.

The ticket is also incorrectly sorted into the "unknown" folder in the ticket.db image for some reason.

**Comment by d0k3** [comment]
I've got an idea what goes wrong here... I'd need to have a look at the ticket. Can you send this to me? Please don't do so openly, but in a pm instead (gbatemp or IRC).

**Comment by enderghast13** [comment]
Okay, I'll make a gbatemp account later today and send it. I can't right now.

**Comment by enderghast13** [comment]
Actually, I can't send pms with a new account so could you send me a pm and maybe I could respond to it? Does gbatemp allow that? https://gbatemp.net/members/enderghast13.425018/

**Comment by ihaveamac** [comment]
new users can't send private messages until about 10 posts or so. either d0k3 needs to start one or you can do it on IRC.

**Comment by enderghast13** [comment]
I don't know how to sign up for irc.

**Comment by ihaveamac** [comment]
you don't have to, just join #cakey on freenode.

**Comment by enderghast13** [comment]
Okay I think I figured it out now, thanks.

**Comment by enderghast13** [comment]
Actually I have no clue how to use an IRC client so...

**Comment by d0k3** [comment]
Timezones ;). I've been asleep, @enderghast13. I sent you a pm on GBAtemp, and I'm also back online on IRC.

**Comment by d0k3** [comment]
Okay, you actually purchased that title, and you'd expect a legit ticket on your system, right? The ticket I saw was corrupted.

Can you try again, with this build of GM9, and let me know if that changes something for you?
https://transfer.sh/mn7yK/GodMode9-20170608-121415.zip

**Comment by enderghast13** [comment]
When I tried this build the ticket did not even display within the ticket.db image in any folder. However, I was able to fix my original problem by first deleting the title's ticket in FBI and visiting its eshop page to have it redownload the ticket. This ticket's titles decrypted correctly and was sorted into the e shop folder correctly, so I guess it was an issue on my end all along. The new ticket displayed in the hex editor just like a normal ticket instead of having a bunch of garbage towards the end.

I can also confirm that I had legitimately purchased DON'T CRASH GO.
